cmd/admin-rpc-client.go

```diff
@@ -88,6 +90,28 @@ func (rc remoteAdminClient) ReInitDisks() error {
 	return rc.Call("Admin.ReInitDisks", &args, &reply)
 }
 
+// Uptime - Returns the uptime of this server. Timestamp is taken
+// after object layer is initialized.
+func (lc localAdminClient) Uptime() (time.Duration, error) {
+	if globalBootTime.IsZero() {
+		return time.Duration(0), errServerNotInitialized
+	}
+
+	return time.Now().UTC().Sub(globalBootTime), nil
```

Comment:
Can you add a notice that this is sometimes buggy but will be automatically fixed in go 1.9 ? https://github.com/golang/go/issues/12914